docs/Users_Guide/load_data.rst

```diff
@@ -2,332 +2,694 @@
+
+    .. literalinclude:: ../../METdbLoad/test/Examples/example_load_specification.xml
+
+.. dropdown:: This is another **example XML specification file** that is **valid**:
+
+    For specifying a list of input data files using the load_files element:
+
+    .. literalinclude:: ../../METdbLoad/test/Examples/example_load_specific_files.xml
+
+
+
+.. dropdown:: This is the **XML schema** that is used to **validate the XML specification file**:
+
+    .. literalinclude:: ../../METdbLoad/ush/load_specification_schema.xsd
+
+Create the XML Specification File
 ===================================
 
-Copy the *data_loading_config.yaml* file to a secure location in your workspace, as this file will contain the username
-and password to the database. **Do not put this file where it can be read by anyone who should not have access to this
-information.**
+The XML specification file contains the database connection information, data file location, and instructions for
+loading the data into the database.
+
+
+Create your own XML specification file by copying the example specification file
+*METdataio/METdbLoad/test/Examples/example_load_specification.xml* file to a
+location in your workspace.
+
+**Do not save this XML specification file where it can be read by anyone who should not have access to this information,
+as this file contains the database password.**
 
 .. code-block:: ini
 
-   cp data_loading_config.yaml /path-to-your-dir/
+   cp $METDATAIO_HOME/METdbLoad/test/Examples/example_load_specification.xml  path-to-your-dir/load_specification.xml
 
-Replace the *path-to-your-dir* with the actual path to where this file is to be saved.
 
-Change directory to the location where the *data_loading_config.yaml* file was copied.
+- $METDATAIO is the path to the location of the cloned or forked METdataio source code.
 
-Open the data_loading_config.yaml file:
+- Replace the *path-to-your-dir* with the actual path to where this file is to be saved.
 
-.. literalinclude:: ../../METdbLoad/sql/scripts/data_loading_config.yaml
+Change directory to the location where the *example_load_specification.xml* file was copied.
 
-Update the database information with information relevant to the database you are using:
+.. code-block:: ini
 
-  * dbname
+   cd path-to-your-dir/load_specification.xml
 
-  * username
+- replace *path-to-your-dir* with the full path where you are saving the XML specification file
 
-  * password
+Using a text editor of your choice, make the necessary edits
+to the required elements and delete any optional, unused/irrelevant elements, based on the explanation below (click to
+expand). Remember to update the username and password that is applicable to your database.
 
-  * host
+.. dropdown:: The following is an explanation of the required and optional elements and any limitations
 
-  * port
+  *These are element names. The XML angle brackets (<>) as seen in the XML specification file are omitted for simplicity*
 
+  **!!!NOTE!!!**
```

Comment:
```suggestion
  ..note::
```
See "Notes" in our documentation for more info: https://metplus.readthedocs.io/en/develop/Contributors_Guide/documentation.html#notes
It seems like a good place to use it here.